plugins/nodes/src/js/pages/nodes-overview/HostsPageContent.js

```diff
@@ -95,7 +95,7 @@ class HostsPageContent extends React.PureComponent {
     const { regions, zones } = this.state.defaultFilterData;
 
     const query = decodeURIComponent(
-      props.location.query["filterExpression"] || ""
+      encodeURIComponent(props.location.query["filterExpression"] || "")
```

Comment:
Do we need to `decodeURIComponent`? Right now it looks like we're encoding and decoding right away.

Reply:
I think we need it, because without it we will type one thing, but we could be actually filtering for another. For example, if we enter "%", we will be filtering for "%25".

Reply:
I mean we're doing `decodeURIComponent(encodeURIComponent(x))` which should be just `x`

Reply:
Yes, but if we don't encode it, it breaks for some symbols, like "%". And if we don't decode it, we could type "%" and it would actually be searching for "%25".

Reply:
I replaced the whole thing with `const query = props.location.query["filterExpression"] || "";` and nothing broke. Am I missing a case here?

Reply:
Hm, I think you are right, good catch. It seems we introduced the decode a long time ago and don't need it anymore.